.github/workflows/continuous-integration.yml

```diff
@@ -192,7 +192,6 @@ jobs:
           set -o pipefail && \
           NSUnbufferedIO=YES xcodebuild \
             -scheme aws-sdk-swift-protocol-tests-Package \
-            -testPlan ProtocolTestPlan \
```

Comment:
This test plan is deleted since it is no longer necessary.  See the companion aws-sdk-swift PR: https://github.com/awslabs/aws-sdk-swift/pull/1784